Point_set_processing_3/examples/Point_set_processing_3/property_map.cpp

```diff
@@ -156,5 +154,27 @@ int main()
       std::cout << points[i]  << std::endl;
     }     
   }
+
+  #ifndef CGAL_CFG_NO_CPP0X_TUPLE
+  // same test with std::tuple
+  {
```

Comment:
It is indeed not ideal but there is nothing in test and the exemple is not inlined in the doc.